### PR TITLE
Add interactive setup of rules.

### PR DESCRIPTION
**Pull request by elibon99**

### CODE REVIEW

File: bark_core/signatures/commands/approve_cmd.py [hunk 1]
```diff
@@ -69,56 +66,44 @@ def approve(ctx, commit, gpg_key_id, ssh_key_path):
...
     repo = project.repo
 
+    key = None
+
     if not gpg_key_id and not ssh_key_path:
-        key = get_key_from_git(repo)
+        config = repo.config
+        if "gpg.format" in config and "user.signingkey" in config:
+            identifier = config["user.signingkey"]
+            type = config["gpg.format"]
```
Comment (dainnilsson) [inline bark_core/signatures/commands/approve_cmd.py]: If `gpg.format` isn't set, does git default to "openpgp"? If yes, shouldn't this do so as well?

File: bark_core/signatures/util.py [hunk 2]
```diff
@@ -1,18 +1,53 @@
...
 import warnings
+import os
+import click
 
 warnings.filterwarnings("ignore")
 
 
 class Pubkey:
     def __init__(self, pubkey: str) -> None:
+        self.bytes = pubkey
```
Comment (dainnilsson) [inline bark_core/signatures/util.py:22]: Type of pubkey is `str`, but this property is named `bytes`, seems wrong

File: bark_core/signatures/commands/approve_cmd.py (hunk 1 above)
Comment (elibon99) [reply bark_core/signatures/commands/approve_cmd.py]: It looks like it defaults to "openpgp". I just pushed a commit that resolves this.

File: bark_core/signatures/util.py (hunk 2 above)
Comment (elibon99) [reply bark_core/signatures/util.py:22]: I agree, `pubkey` should be of type `bytes`! Pushed a commit that resolves this.

File: bark_core/signatures/commands/approve_cmd.py [hunk 3]
```diff
@@ -69,56 +66,44 @@ def approve(ctx, commit, gpg_key_id, ssh_key_path):
...
     repo = project.repo
 
+    key = None
+
     if not gpg_key_id and not ssh_key_path:
-        key = get_key_from_git(repo)
+        config = repo.config
+        if "user.signingkey" in config:
+            identifier = config["user.signingkey"]
+            type = config["gpg.format"] if "gpg.format" in config else "openpgp"
```
Comment (dainnilsson) [inline bark_core/signatures/commands/approve_cmd.py:75]: Can't this be written as just `type = config.get("gpg.format", "openpgp")`?

**Review by dainnilsson — APPROVED** [review]

File: bark_core/signatures/commands/approve_cmd.py (hunk 3 above)
Comment (elibon99) [reply bark_core/signatures/commands/approve_cmd.py:75]: I tried, but `config` does not expose the `get` method. It is not a `dict`.

**Pull request merged**